convert.py

```diff
@@ -1036,7 +1036,8 @@ def load_some_model(path: Path) -> ModelPlus:
     # Be extra-friendly and accept either a file or a directory:
     if path.is_dir():
         # Check if it's a set of safetensors files first
-        files = list(path.glob("model-00001-of-*.safetensors"))
+        globs = ["model-00001-of-*.safetensors", "model.safetensors"]
```

Comment:
Why not `*.safetensors`? That's the common approach.

Reply:
I thought the same thing, but that _could_, under _some_ absurd circumstances cause problems. We never know what people do with their stuff... Might wait for someone with another option though

Reply:
@AlpinDale The glob looks like it's deliberately trying to target the first part of the set with `model-00001-of-*`. If it was just `*.safetensors` then you could get `model-99999-of-99999.safetensors` which is probably not what you want to load.

Reply:
Python indexes the files alphabetically when using a glob, so that is a non-issue. I'm simply pointing out that this way of doing it is unconventional and I've not seen any other project do this.

Reply:
> Python indexes the files alphabetically when using a glob

I'm pretty sure that's not the case. The documentation doesn't even mention order: https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob

Note also that their examples are like `sorted(Path('.').glob('*.py'))` which would be redundant if it was already sorted.

> I'm simply pointing out that this way of doing it is unconventional 

That may be the case, but your proposed change would break it. There's actually a

```python
1046   │         if len(files) > 1:
1047   │             raise # ...
```

a couple lines down. This is specifically supposed to pull in the first file of the set, not all of them.